File: Modul 6/Python/PRAK603-2310817310005-MuhammadNurwahyudiAdhitama.py
import numpy as np

baris1, baris2 = map(int, input("").split())

if (baris1 == baris2) :

    E_ARRAY1 = list(map(int,input("").split()))
    E_ARRAY2 = list(map(int,input("").split()))

    arr1 = np.array(E_ARRAY1).reshape(1,baris1)
    arr2 = np.array(E_ARRAY2).reshape(1,baris2)

    i = 0
    
    while i < baris1 and i < baris2 :
        HASIL = E_ARRAY1[i] * E_ARRAY2[i]
        print(HASIL, end = " ")
        i += 1

else :
    print("Jumlah tidak sama")

File: Modul 6/C/PRAK603-2310817310005-MuhammadNurwahyudiAdhitama.c
#include <stdio.h>

int main() {
    int baris1, baris2;

    scanf("%d %d", &baris1, &baris2);

    
    if (baris1 == baris2) {
        int arr1[baris1];
        int arr2[baris2];

        for (int i = 0; i < baris1; i++) {
            scanf("%d", &arr1[i]);
        }

        for (int i = 0; i < baris2; i++) {
            scanf("%d", &arr2[i]);
        }

        for (int i = 0; i < baris1 && i < baris2; i++) {
            int hasil = arr1[i] * arr2[i];
            printf("%d ", hasil);
        }
        printf("\n");
    } else {
        printf("Jumlah elemen tidak sama.\n");
    }

    return 0;
}
